pkg/beacon/relay/groupselection/groupselection.go

```diff
@@ -87,192 +92,188 @@ func CandidateToNewGroup(
 }
 
 func startTicketSubmission(
```

Comment:
Let's maybe rename it to `submitTickets`. `startTicketSubmission` suggests there is some other function that will end the ticket submission.